src/app/components/repeater/index.html

```diff
@@ -61,6 +61,13 @@
     >
     Indicates whether the repeater item's checkbox is selected. When users select the repeater item, the specified property on your model is updated accordingly. This property accepts <sky-code>boolean</sky-code> values.
     </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="reorderable"
```

Comment:
`reorderable` is both an input property for the repeater component and the repeater item component. Here, we've only documented it on the repeater item component, but all our demos/tests show it only used on the parent repeater component. This should probably be moved to a repeater component property.
@Blackbaud-TrevorBurch might need to confirm this.

Reply:
Thanks for catching that. I dug into the source file for the repeater item component because that's what I noticed on the PR. I'll check the source file for the repeater component as well. Thanks!

Reply:
I updated the docs to reflect the second property. Are the descriptions corrrect?